packages/node_modules/@webex/plugin-meetings/src/reachability/index.js

```diff
@@ -57,7 +59,7 @@ export default class Reachability extends StatelessWebexPlugin {
     this.setup();
 
     // Remove stored reachability results to ensure no stale data
-    if (window.localStorage && window.localStorage.removeItem) {
+    if (window?.localStorage?.removeItem) {
```

Comment:
👍 Nice, when did we get the upgrade to support this?

Reply:
A couple months ago! Very exciting!